draft-ietf-ppm-dap.md

```diff
@@ -1391,124 +1317,230 @@ Otherwise, `out` is the pair `(prep_state, prep_msg)`.
+
+* The opaque, VDAF-specific aggregation parameter provided during the collection
+  flow (`agg_param`),
+
+  [[OPEN ISSUE: Check that this handling of `agg_param` is appropriate when the
+  definition of Poplar is done.]]
+
+* Information used by the Aggregators to determine how to aggregate each report:
+
+    * For fixed_size tasks, the Leader specifies a "batch ID" that determines
+      the batch to which each report for this aggregation job belongs.
+
+      [OPEN ISSUE: For fixed_size tasks, the Leader is in complete control over
+      which batch a report is included in. For time_interval tasks, the Client
+      has some control, since the timestamp determines which batch window it
+      falls in. Is this desirable from a privacy perspective? If not, it might
+      be simpler to drop the timestamp altogether and have the agg init request
+      specify the batch window instead.]
+
+  The indicated query type MUST match the task's query type. Otherwise, the
+  Helper MUST abort with error "queryMismatch".
+
+* The sequence of report shares to aggregate. The `encrypted_input_share` field
+  of the report share is the `HpkeCiphertext` whose index in
+  `Report.encrypted_input_shares` is equal to the index of the aggregator in the
+  task's `aggregator_endpoints` to which the AggregateInitializeReq is being
+  sent.
+
+#### Required methods
+
+##### PUT `/tasks/{task-id}/aggregation_jobs/{aggregation-job-id}` {#aggregation-job-put}
+
+`task-id` is the ID of the task that the aggregation job belongs to.
+`aggregation-job-id` is an `AggregationJobId` chosen by the leader to uniquely
+identify an aggregation job across the rounds of the aggregation sub-protocol.
+
+The Leader begins an aggregation job by choosing a set of candidate reports that
+pertain to the same DAP task and compiling them into an `AggregationJob` (see
+{{aggregation-job-representation}}. The Leader can prepare many reports in
+parallel by concurrently creating multiple aggregation jobs. Creating
+aggregation jobs is how the Leader transmits report shares to the helper, so
+each `PrepareStep.prepare_step_state` MUST be set to `PrepareStepState.start`
+and the `PrepareStep` MUST contain a `ReportShare`. `AggregationJob.round` MUST
+be 0.
+
+If successful, the Helper's responds with a `struct AggregationJob` whose
+`round` field is 1 and whose `prepare_steps` is a list of `struct PrepareStep`s
+in state `continued` if initialization succeeded or errors if any report was
+invalid.
+
+##### POST `/tasks/{task-id}/aggregation_jobs/{aggregation-job-id}` {#aggregation-job-post}
+
+The Leader drives the continuation of input preparation using POST requests to
+aggregation job resources (see {{agg-continue-flow}}).
+
+###### DELETE `/tasks/{task-id}/aggregation_jobs/{aggregation-job-id}`
+
+Once an aggregation job is deleted, the Helper MAY abandon it and discard all
+state related to it (e.g., report shares, prepare messages, preparation state).
```

Comment:
Is the Helper required to keep the agg job state around until it is DELETEd? As a deployer, I don't want to have to count on my Leader to manage my state responsibly.